plugins/osx/osx.plugin.zsh

```diff
@@ -161,6 +161,15 @@ function itunes() {
 		vol)
 			opt="set sound volume to $1" #$1 Due to the shift
 			;;
+		playing)
+			local state=`osascript -e 'tell application "iTunes" to player state as string'`
+			if [ "$state" = "playing" ]; then
+				currenttrack=`osascript -e 'tell application "iTunes" to name of current track as string'`
+				currentartist=`osascript -e 'tell application "iTunes" to artist of current track as string'`
+				echo "Listening to %F{yellow}$currenttrack%f by %F{yellow}$currentartist%f";
```

Comment:
The `%F` sequences only work with `print -P` not with regular `echo`.

<img width="853" alt="screen shot 2015-09-16 at 3 41 04 pm" src="https://cloud.githubusercontent.com/assets/2618447/9916215/5a7940b0-5c89-11e5-8ea0-4b8f8c7449f3.png">